package/src/components/RegionInput/v1/RegionInput.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component, Fragment } from "react";
+import PropTypes from "prop-types";
+import { withComponents } from "@reactioncommerce/components-context";
+import { CustomPropTypes } from "../../../utils";
+
+class RegionInput extends Component {
+  static isFormInput = true;
```

Comment:
It's probably better to remove `isFormInput` from here because this component doesn't implement the full form input API. It renders either `Select` or `TextInput`, both of which are valid form inputs, so ReactoForm will just drop down a level and get the value from those.

`WrappedRegionInput.isFormInput = true;` below is also not necessary